ipam/allocator.go

```diff
@@ -416,32 +416,32 @@ func (a *Allocator) getPredefinedPool(as string, ipV6 bool) (*net.IPNet, error)
 }
 
 // RequestAddress returns an address from the specified pool ID
-func (a *Allocator) RequestAddress(poolID string, prefAddress net.IP, opts map[string]string) (*net.IPNet, map[string]string, error) {
+func (a *Allocator) RequestAddress(poolID string, prefAddress net.IP, opts map[string]string) (*net.IPNet, []string, []string, map[string]string, error) {
```

Comment:
I have a personal dislike towards overusing the multiple-return values of golang.
It just makes the code hard to read and easy to make silly mistakes.
@aboch wdyt ? do you have a better suggestion ?